integrations/whatsapp/src/misc/wizard.ts

```diff
@@ -23,9 +30,19 @@ export const handleWizard = async (req: Request, client: bp.Client, ctx: bp.Cont
         { display: 'No', type: 'secondary', action: 'CLOSE_WINDOW' },
       ],
     })
+  } else {
```

Comment:
This else condition is just to be safe. Duplicate events don't hurt at this scale, and this helps get a better picture of whats going on.

Reply:
Yes, and this would also help checking if a new wizard step was added by Meta and somehow users are ending up in it.